src/dbm/dbm_internal.h

```diff
@@ -5,8 +5,13 @@
 /*  SPDX-License-Identifier: BSD-3-Clause                                     */
 /*----------------------------------------------------------------------------*/
 
-#ifndef DBM_MULTIPLY_INTERNAL_H
-#define DBM_MULTIPLY_INTERNAL_H
+#ifndef DBM_INTERNAL_H
+#define DBM_INTERNAL_H
+
+/*******************************************************************************
+ * \brief Returns the larger of two given integer (missing from the C standard)
+ ******************************************************************************/
+#define DBM_MAX(X, Y) ((X) > (Y) ? (X) : (Y))
```

Comment:
Could we turn this into a function again? Performance-wise it should be the same, but it's easier to read and has additional type-safety.

Reply:
Will do. I was about to implement the respective function yet again in another module, then I put it into the header, then I realized, I need another type-combination not just `int` (but rather unsigned long int or size_t). Currently, it's only used as int-flavor. I will turn it back into a function.

Reply:
( btw, I removed the author name since I did not want to "let you write that macro". I will bring it back also ;-)